terraform/iam_user.tf

```diff
@@ -1,6 +1,10 @@
-# The users being created
+locals {
+  test_user_name = "test-ansible-role-nessus"
```

Comment:
Test user at last!

More seriously, we should be consistent with our nomenclature.  I think "test user" makes sense here, but as long as it's consistent I won't complain.

Reply:
Consistency has been achieved in https://github.com/cisagov/ansible-role-nessus/pull/8/commits/cdf958e70560f8ec8e461e8b7c7b1688e46d61df.